### PR TITLE
Improve limit plot

### PR DESCRIPTION
**Pull request by cg-laser**

### CODE REVIEW

**Comment by anelles** [comment]
@clark2668 I saw the ARA 2023 projection, thanks for adding. One question: Does this include that no one can go to South Pole this year and fix ARA 2 - 4 this year? Or what is the assumption?

**Comment by clark2668** [comment]
@anelles , thanks--probably not. I'm working on refining the assumptions now.

**Comment by cg-laser** [comment]
now that the ICRC is over and the plots are final, I think we can merge this one.

**Comment by clark2668** [comment]
I think this is mostly good, but a few quick points might help things:

1. Can we converge on having a single `E2_fluxes.py` file? I suspect this means removing `E2_fluxes.py` and `E2_fluxes2.py` and renaming `E2_fluxes3.py` so that it's the single remaining version.
2. Can we remove unnecessary comments from `E2_fluxes3.py`? The HESE 6 years is all commented out for some reason?  And there are lines commented out in some places, e.g. [here](https://github.com/nu-radio/NuRadioMC/blob/improve-limit-plot/NuRadioMC/examples/Sensitivities/E2_fluxes3.py#L175) and [here](https://github.com/nu-radio/NuRadioMC/blob/improve-limit-plot/NuRadioMC/examples/Sensitivities/E2_fluxes3.py#L209).
3. Can we add the Gen2 review array (the one shown in the proceedings) to the repository, either as a CSV file or inline in the file? I think this is important to ensure reproducibility.

File: NuRadioMC/examples/Sensitivities/E2_fluxes3.py [hunk 1]
```diff
@@ -162,25 +167,64 @@
...
-
-
-def ice_cube_nu_fit(energy, slope=-2.19, offset=1.01):
-    # ApJ slope=-2.13, offset=0.9
+# ice cube nu_mu data points 9.5 years analysis
+nu_mu_data = np.array([[4.64588, -7.69107, -7.87555, -7.5549, ],
+                       [5.44266, -7.95022, -8.06881, -7.85359, ],
+                       [6.25755, -8.51245, -8.8287, -8.29283],
+                       [7.29276, -8.40264, 0, 0]])
+# nu_mu_data[:, 2] = 10 ** nu_mu_data[:, 1] - 10 ** (nu_mu_data[:, 1] - nu_mu_data[:, 2])
```
Comment (clark2668) [inline NuRadioMC/examples/Sensitivities/E2_fluxes3.py:175]: It would be good to cleanup commented out lines like this.

**Review by clark2668 — CHANGES_REQUESTED** [review]
I'll put this here also (so it exists in the review document):

I think this is mostly good, but a few quick points might help things:

- Can we converge on having a single E2_fluxes.py file? I suspect this means removing E2_fluxes.py and E2_fluxes2.py and renaming E2_fluxes3.py so that it's the single remaining version.
- Can we remove unnecessary comments from E2_fluxes3.py? The HESE 6 years is all commented out for some reason? And there are lines commented out in some places. I marked one of them.
- - Can we add the Gen2 review array (the one shown in the proceedings) to the repository, either as a CSV file or inline in the file? I think this is important to ensure reproducibility.

**Review by clark2668 — APPROVED** [review]
Approved.

**Pull request merged**